chaincode-labs/chaincode-residency/2019-06-28-christian-decker-rendezvous-routing.md

```diff
@@ -0,0 +1,49 @@
+---
+title:  Christian Decker - Rendezvous Routing (2019-06-28)
+transcript_by: 
```

Comment:
Name(s) of transcript author should be included

Reply:
thank you! I've updated